IControlLimits.h is not a self-sufficient header

**Describe the bug**
`IControlLimits.h` is not a [Self-sufficient Header](http://www.drdobbs.com/self-sufficient-headers/184401705). This can create unexpected compilation problems in downstream code, and create problems with build systems such as `bazel`. 

**To Reproduce**
Compile a compilation unit that just includes `iControlLimits.h".
You will get the following error:
~~~
In file included from /home/straversaro/src/tutorial_impedance-control/Module.h:7:0,
                 from /home/straversaro/src/tutorial_impedance-control/main.cpp:1:
/home/straversaro/src/robotology-superbuild/build/install/include/yarp/dev/IControlLimits.h:129:62: error: ‘createVocab’ is not a member of ‘yarp::os’
 constexpr yarp::conf::vocab32_t VOCAB_LIMITS     = yarp::os::createVocab('l','l','i','m');
                                                              ^~~~~~~~~~~
/home/straversaro/src/robotology-superbuild/build/install/include/yarp/dev/IControlLimits.h:130:62: error: ‘createVocab’ is not a member of ‘yarp::os’
 constexpr yarp::conf::vocab32_t VOCAB_VEL_LIMITS = yarp::os::createVocab('v','l','i','m');
                                                              ^~~~~~~~~~~
CMakeFiles/tutorial_impedance-control.dir/build.make:62: recipe for target 'CMakeFiles/tutorial_impedance-control.dir/main.cpp.o' failed
make[2]: *** [CMakeFiles/tutorial_impedance-control.dir/main.cpp.o] Error 1
CMakeFiles/Makefile2:99: recipe for target 'CMakeFiles/tutorial_impedance-control.dir/all' failed
make[1]: *** [CMakeFiles/tutorial_impedance-control.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
~~~

**Expected behavior**
Any compilation unit that includes `iControlLimits.h` should be compilable. 

**Configuration (please complete the following information):**
 - OS: Ubuntu 18.04 
 - yarp version: 3.1
-  compiler: gcc 7.3

--- a/doc/release/v3_1_1.md
+++ b/doc/release/v3_1_1.md
@@ -14,6 +14,10 @@ Bug Fixes
 
 * Fix `write()` in BufferedPort after interrupting-resuming(#1834).
 
+#### YARP_dev
+
+* Fix `IControlLimits.h` not being a self-sufficient header (#1845).
+
 ### Bindings
 
 * Usage of methods that take in input a yarp::sig::Vector in bindings has been

--- a/src/libYARP_dev/include/yarp/dev/IControlLimits.h
+++ b/src/libYARP_dev/include/yarp/dev/IControlLimits.h
@@ -10,6 +10,7 @@
 #ifndef YARP_DEV_ICONTROLLIMITS_H
 #define YARP_DEV_ICONTROLLIMITS_H
 
+#include <yarp/os/Vocab.h>
 #include <yarp/dev/DeviceDriver.h>
 
 /*! \file IControlLimits.h define control board standard interfaces*/

